src/Token/AccessToken.php

```diff
@@ -196,7 +196,7 @@ public function hasExpired()
             throw new RuntimeException('"expires" is not set on the token');
         }
 
-        return $expires < time();
+        return $expires < self::getTimeNow();
```

Comment:
Should this be `$this->getTimeNow()`, since `getTimeNow()` is an instance method and not a static method?